charts/newrelic-pixie/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart for the New Relic Pixie integration.
 name: newrelic-pixie
-version: 2.1.5
-appVersion: 2.1.4
+version: 2.2.1
```

Comment:
Shouldn't this chart `version` be bumped to `2.1.6`?

Reply:
Seems like only `appVersion` would need to match the tag of the latest image from this [line](https://github.com/newrelic/helm-charts/blob/fe25b60288bceaaba3c5459fc6a24ece8e3c6028/charts/newrelic-pixie/templates/job.yaml#L56) in the template.

Reply:
Updated to 2.1.6